src/onegov/translator_directory/utils.py

```diff
@@ -207,12 +210,41 @@ def geocode_translator_addresses(
     return trs_total, total, geocoded, skipped, coords_not_found
 
 
-def nationality_choices(locale: str | None) -> list['_Choice']:
+def nationality_choices(locale: str | None) -> list[tuple[str, str]]:
     assert locale
-    _locale = Locale.parse(locale)
-    nationalities = sorted((
-        (code, name) for code, name in
-        _locale.territories.items() if len(code) == 2
-    ), key=itemgetter(1))
+
+    nationalities = [(code, name) for code, name in
+                     country_code_to_name(locale).items()]
+
+    # pin common countries on top of the list
     nationalities.insert(0, ('', ''))  # add empty choice
+    nationalities.insert(1, nationalities.pop(
+        nationalities.index(('CH', 'Schweiz'))))
+    nationalities.insert(2, nationalities.pop(
+        nationalities.index(('DE', 'Deutschland'))))
+    nationalities.insert(3, nationalities.pop(
+        nationalities.index(('FR', 'Frankreich'))))
+    nationalities.insert(4, nationalities.pop(
+        nationalities.index(('IT', 'Italien'))))
+    nationalities.insert(5, nationalities.pop(
+        nationalities.index(('AT', 'Österreich'))))
+    nationalities.insert(6, ('', '------'))
```

Comment:
```suggestion
    country_names = country_code_to_name(locale)
    pinned = ('CH', 'DE', 'FR', 'IT', 'AT', 'LI')
    nationalities = [(code, name) for code, name in
                     country_names.items() if code not in pinned]
    # pin common countries on top of the list
    nationalities.insert(0, ('', '------'))  # add divider
    for code in reversed(pinned):
        nationalities.insert(0, (code, country_names.get(code, code)))
    nationalities.insert(0, ('', ''))  # add empty choice
```
Your code breaks if the locale isn't German, this should be more robust

Reply:
Yes, have it on my todo list, but l like your implementation!